cognite/client/_api/entity_matching.py

```diff
@@ -0,0 +1,220 @@
+
+        Returns:
+            EntityMatchingModelList: List of models."""
+        filter = {utils._auxiliary.to_camel_case(k): v for k, v in (filter or {}).items() if v is not None}
+        # NB no pagination support yet
+        models = self._post(self._RESOURCE_PATH + "/list", json={"filter": filter}).json()["items"]
+        return EntityMatchingModelList(
+            [self._LIST_CLASS._RESOURCE._load(model, cognite_client=self._cognite_client) for model in models]
+        )
+
+    def list_jobs(self) -> EntityMatchingModelList:
+        """List jobs
+
+        Returns:
+            EntityMatchingModelList: List of jobs."""
+        return EntityMatchingModelList(
+            [
+                self._LIST_CLASS._RESOURCE._load(model, cognite_client=self._cognite_client)
+                for model in self._get(self._RESOURCE_PATH + "/jobs").json()["items"]
+            ]
+        )
+
+    def delete(self, id: Union[int, List[int]] = None, external_id: Union[str, List[str]] = None) -> None:
+        """Delete models
+
+        Args:
+            id (Union[int, List[int]): Id or list of ids
+            external_id (Union[str, List[str]]): External ID or list of external ids"""
+        self._delete_multiple(ids=id, external_ids=external_id, wrap_ids=True)
+
+    def fit(
+        self,
+        sources: List[Union[Dict, CogniteResource]],
+        targets: List[Union[Dict, CogniteResource]],
+        true_matches: List[Union[Dict, Tuple[Union[int, str], Union[int, str]]]] = None,
+        match_fields: List[Tuple[str, str]] = None,
+        feature_type: str = None,
+        classifier: str = None,
+        ignore_missing_fields: bool = False,
+        name: str = None,
+        description: str = None,
+        external_id: str = None,
+    ) -> EntityMatchingModel:
+        """Fit entity matching model.
+        Note: All users on this CDF subscription with assets read-all capability in the project, are able to access the data sent to this endpoint.
+
+        Args:
+            sources: entities to match from, should have an 'id' field. Tolerant to passing more than is needed or used (e.g. json dump of time series list)
+            targets: entities to match to, should have an 'id' field.  Tolerant to passing more than is needed or used.
+            true_matches: Known valid matches given as a list of dicts with keys 'sourceId', 'sourceExternalId', 'sourceId', 'sourceExternalId'). If omitted, uses an unsupervised model.
+             A tuple can be used instead of the dictionary for convenience, interpreted as id/externalId based on type.
+            match_fields: List of (from,to) keys to use in matching. Default in the API is [('name','name')]
+            feature_type (str): feature type that defines the combination of features used, see API docs for details.
+            classifier (str): classifier used in training. Currently undocumented in API.
```

Comment:
is documented in the most recent pr?

Reply:
should be

Reply:
then i would remove "Currently undocumented in API."